config/opal_mca.m4

```diff
@@ -767,12 +767,12 @@ AC_DEFUN([MCA_PROCESS_COMPONENT],[
     else
         if test "$2" = "common"; then
             # Static libraries in "common" frameworks are installed, and
-            # therefore must obey the $FRAMEWORK_LIB_PREFIX that was
+            # therefore must obey the $FRAMEWORK_LIB_NAME that was
```

Comment:
This functionality is all for IBM; can you make sure that IBM is all aligned that this is the right thing to do?  You're fairly fundamentally changing how libraries will be shipped.

Reply:
This doesn't really change anything, it's more cleanup. If needed/desired the opal lib name could still be changed quite easily. That said, I don't believe it is currently used, I'll verify though. @gpaulsen @jjhursey do you agree?